src/DocRaptor.php

```diff
@@ -176,26 +188,36 @@ public function fetchDocument($filename = false)
             }
 
             if (!empty($this->document_content)) {
-                $fields['doc[document_content]'] = urlencode($this->document_content);
+                $fields['doc[document_content]'] = $this->document_content;
             } else {
-                $fields['doc[document_url]'] = urlencode($this->document_url);
+                $fields['doc[document_url]'] = $this->document_url;
             }
 
-            $fields_string = http_build_query($fields);
+            $queryString = http_build_query($fields);
```

Comment:
I was planning to leave snake_case only for fields which are used by the API. I otherwise never use snake_case when writing PHP. This way all normal vars would be camel and the ones that have a mapping with the API would stay snake. ?